data/brands/amenity/restaurant.json

```diff
@@ -43,6 +43,17 @@
     }
   },
   "items": [
+    {
+      "displayName": "Genghis Grill",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "restaurant",
```

Comment:
Not to throw more confusion into the works, but per the definition in https://github.com/osmlab/name-suggestion-index/wiki/Judge-Case#restaurant-or-fastfood `amenity=fast_food` would probably fit this better, especially for the post-2021 design.